app/leave-app/backend/constants.bal

```diff
@@ -0,0 +1,70 @@
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+import ballerina/lang.regexp;
+
+public const JWT_CONTEXT_KEY = "JWT_CONTEXT_KEY";
+public const TOTAL_LEAVE_TYPE = "total";
+public const TOTAL_EXCLUDING_LIEU_LEAVE_TYPE = "totalExLieu";
+public const decimal DAY_IN_SECONDS = 86400;
+public final readonly & EmployeeStatus[] DEFAULT_EMPLOYEE_STATUSES = [EmployeeStatusActive, EmployeeStatusMarkedLeaver];
+public final map<string> & readonly TIMEZONE_OFFSET_MAP = {
+    "Australia": "+10:00",
+    "Brazil": "-03:00",
+    "Canada": "-05:00",
+    "US": "-07:00",
+    "Sri Lanka": "+05:30",
+    "UK": "+01:00",
+    "Argentina": "-03:00",
+    "Mexico": "-06:00",
+    "Columbia": "-05:00",
+    "Saudi Arabia": "+03:00",
+    "Germany": "+01:00",
+    "Greece": "+02:00",
+    "France": "+01:00",
+    "Netherland": "+01:00",
+    "Spain": "+02:00",
+    "India": "+05:30",
+    "New Zealand": "+12:00",
+    "Singapore": "+08:00"
+};
+
+// Errors
+public const ERR_MSG_CANCEL_LEAVE = "Error occurred when fetching cancelled leave!";
+public const ERR_MSG_EFFECTIVE_DAYS_FAILED = "Error when getting effective days!";
+public const ERR_MSG_EMPLOYEE_LOCATION_RETRIEVAL_FAILED = "Error occurred while retrieving employee location!";
+public const ERR_MSG_FORBIDDEN_TO_NON_LEADS = "You have not been assigned as a lead/manager to any employee!";
+public const ERR_MSG_HOLIDAYS_RETRIEVAL_FAILED = "Error occurred while retrieving holidays!";
+public const ERR_MSG_INVALID_DATE_FORMAT = "Invalid date. Date string should be in ISO 8601 format!";
+public const ERR_MSG_LEGALLY_ENTITLED_LEAVE_RETRIEVAL_FAILED =
+    "Error occurred while retrieving legally entitled leaves!";
+public const ERR_MSG_LEAVE_ENTITLEMENT_RETRIEVAL_FAILED = "Error occurred while retrieving leave entitlement!";
+public const ERR_MSG_LEAVE_IN_INVALID_STATE = "Leave is in an invalid state!";
+public const ERR_MSG_LEAVE_OVERLAPS_WITH_EXISTING_LEAVE = "Leave overlaps with existing leave(s)!";
+public const ERR_MSG_LEAVES_RETRIEVAL_FAILED = "Error occurred while retrieving leaves!";
+public const ERR_MSG_LEAVE_SHOULD_BE_AT_LEAST_ONE_WORKING_DAY =
+    "Leave requests should contain at least one working day!";
+public const ERR_MSG_LEAVE_SHOULD_BE_AT_LEAST_ONE_WEEKDAY = "Leave requests should contain at least one weekday!";
+public const ERR_MSG_NO_JWT_TOKEN_PRESENT = "x-jwt-assertion header does not exist!";
+public const ERR_MSG_END_DATE_BEFORE_START_DATE = "End date cannot be before start date!";
+public const ERR_MSG_UNAUTHORIZED_VIEW_LEAVE = "You are not authorized to view the requested leaves!";
+public const ERR_MSG_INVALID_WSO2_EMAIL = "Input email is not a valid WSO2 email address!";
+
+// Regex
+final regexp:RegExp & readonly REGEX_DATE_YYYY_MM_DD = re `^\d{4}-\d{2}-\d{2}`;
```

Comment:
Shall we use `string:RegExp` instead of `regexp:RegExp` if the current Ballerina version supports this? If not, shall we upgrade the Ballerina version also?
https://central.ballerina.io/ballerina/lang.regexp/latest